Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -3323,7 +3324,7 @@ bool HTMLInputElement::suffering_from_being_missing() const
             if (is_in_same_radio_button_group(*this, element)) {
                 if (element.checked())
                     has_checkedness_false_for_all_elements_in_group = false;
-                if (has_attribute(HTML::AttributeNames::required))
+                if (element.has_attribute(HTML::AttributeNames::required))
```

Comment:
Commit message: could you try to avoid "fix" in the title and instead describe what you changed? E.g.

LibWeb: Check all radio buttons in group for required attribute